src/img.rs

```diff
@@ -92,9 +92,14 @@ fn get_exif_dimensions(exif: &Exif) -> Option<ImageDimensions> {
 }
 
 fn get_date_time(value: Option<&Field>) -> Option<chrono::DateTime<Local>> {
-    Local
-        .datetime_from_str(get_str(value)?, "%Y:%m:%d %H:%M:%S")
-        .ok()
+    let str = get_str(value)?;
+    match Local.datetime_from_str(str, "%Y:%m:%d %H:%M:%S") {
```

Comment:
matching the result and converting it to an error can be correctly done like this. 👍 

There is a method to do that for you, `Result::ok`, so you can just call `.ok()`. I did that accidentally in a previous commit I pushed.